src/editor.js

```diff
@@ -0,0 +1,154 @@
+
+const ExportTheme = () => {
+	const { createErrorNotice } = useDispatch( noticesStore );
+	const [ theme, setTheme ] = useState( {
+		name: '',
+		description: '',
+		uri: '',
+		author: '',
+		author_uri: '',
+	} );
+
+	useSelect( ( select ) => {
+		const themeData = select( 'core' ).getCurrentTheme();
+		setTheme( {
+			name: themeData.name.raw,
+			description: themeData.description.raw,
+			author: themeData.author.raw,
+			author_uri: themeData.author_uri.raw,
+			theme_uri: themeData.theme_uri.raw,
+		} );
+	}, [] );
+
+	const handleSubmit = () => {
+		fetch( '/wp-json/create-block-theme/v1/export', {
+			method: 'POST',
+			headers: {
+				'Content-Type': 'application/json',
+			},
+			body: JSON.stringify( theme ),
+		} )
+			.then( ( response ) => response.blob() )
+			.then( ( blob ) => {
+				const url = URL.createObjectURL( blob );
+				window.location.href = url;
+			} )
+			.catch( ( error ) => {
+				const errorMessage =
+					error.message && error.code !== 'unknown_error'
+						? error.message
+						: __(
+								'An error occurred while attempting to export the theme.'
+						  );
+				createErrorNotice( errorMessage, { type: 'snackbar' } );
+			} );
+	};
+
+	return (
+		<PanelBody>
+			<Heading>{ __( 'Export', 'create-block-theme' ) }</Heading>
+			<VStack>
+				<Text variant="muted">
+					{ __(
+						"Make any changes to the current theme's metadata and export as a new theme.",
+						'create-block-theme'
+					) }
+				</Text>
+				<TextControl
+					label={ __( 'Theme name', 'create-block-theme' ) }
+					value={ theme.name }
+					onChange={ ( value ) =>
+						setTheme( { ...theme, name: value } )
+					}
+					placeholder={ __( 'Theme name', 'create-block-theme' ) }
+				/>
+				<TextControl
+					label={ __( 'Theme description', 'create-block-theme' ) }
+					value={ theme.description }
+					onChange={ ( value ) =>
+						setTheme( { ...theme, description: value } )
+					}
+					placeholder={ __(
+						'A short description of the theme',
+						'create-block-theme'
+					) }
+				/>
+				<TextControl
+					label={ __( 'Theme URI', 'create-block-theme' ) }
+					value={ theme.uri }
+					onChange={ ( value ) =>
+						setTheme( { ...theme, uri: value } )
+					}
+					placeholder={ __(
+						'https://github.com/wordpress/twentytwentytwo/',
+						'create-block-theme'
+					) }
+				/>
+				<TextControl
+					label={ __( 'Author', 'create-block-theme' ) }
+					value={ theme.author }
+					onChange={ ( value ) =>
+						setTheme( { ...theme, author: value } )
+					}
+					placeholder={ __(
+						'the WordPress team',
+						'create-block-theme'
+					) }
+				/>
+				<TextControl
+					label={ __( 'Author URI', 'create-block-theme' ) }
+					value={ theme.author_uri }
+					onChange={ ( value ) =>
+						setTheme( { ...theme, author_uri: value } )
+					}
+					placeholder={ __(
+						'https://wordpress.org/',
+						'create-block-theme'
+					) }
+				/>
```

Comment:
This button looks weird!

Reply:
I'm not sure about primary since there is already the "Save" primary button. What do you think of this?

<img width="286" alt="Screenshot 2023-03-29 at 5 26 47 PM" src="https://user-images.githubusercontent.com/5375500/228671880-95ad3836-266c-4c6a-a8fa-933dfdbdec1c.png">

Reply:
WFM